pnpm-lock.yaml

```diff
@@ -500,12 +501,26 @@ importers:
       '@emotion/styled': 11.8.1_0bd19656468de1d4aa39379bbff84910
       '@fontsource/pt-serif': 4.5.3
       '@fontsource/spartan': 4.5.4
+<<<<<<< HEAD
```

Comment:
This merge conflict got added to the lock file.  You can fix by deleting this file and running `pnpm install` in the root again, to regenerate it.